src/__tests__/connection/passwordless/passwordless.test.js

```diff
@@ -51,6 +51,18 @@ describe('passwordless connection', () => {
         });
         expectMockToMatch(mockFns.initNS, 1);
       });
+      it('with mustAcceptTerms:true when opts.mustAcceptTerms is true', () => {
```

Comment:
Awkward statement ... should start with "should"

Reply:
it makes sense if you join the `describe` description with this (that's how the test runner shows it).

`calls initNS with mustAcceptTerms:true when opts.mustAcceptTerms is true`